csrankings-b.csv

```diff
@@ -645,7 +645,7 @@ Bongwon Suh,Seoul National University,http://hcc.snu.ac.kr/wordpress,-nlhtEkAAAA
 Bonita Sharif,University of Nebraska,https://www.shbonita.me,2WeXBokAAAAJ
 Bonnie A. Nardi,Univ. of California - Irvine,http://www.artifex.org/~bonnie,hwc6A1MAAAAJ
 Bonnie Berger,Massachusetts Institute of Technology,https://people.csail.mit.edu/bab,qMOS01AAAAAJ
-Bonnie J. Dorr,University of Maryland - College Park,https://www.cs.umd.edu/people/bdorr,jSjEWosAAAAJ
+Bonnie J. Dorr,University of Florida - Gainesville,https://www.cise.ufl.edu/~bonniejdorr,jSjEWosAAAAJ

```

Comment:
The name of the institution in the database is "University of Florida". Please fix and I will merge.